pkg/handler3/handler.go

```diff
@@ -74,6 +77,14 @@ func init() {
 	mime.AddExtensionType(".gz", mimePbGz)
 }
 
+// unquoteEtag removes the string quotes around an ETag
+func unquoteETag(etag string) string {
```

Comment:
This function has similar semantics to [`strconv.Unquote`](https://pkg.go.dev/strconv#Unquote). Should that be used instead?

Reply:
I'd even suggest we don't prematurely quote the etag so we don't have to untag it later?

Reply:
I'll update the etag so it's not prematurely quoted and use strconv.Quote when we need to